pkg/cmd/label/create/create.go

```diff
@@ -130,5 +132,27 @@ func createLabel(client *http.Client, repo ghrepo.Interface, opts *CreateOptions
 	}
 	requestBody := bytes.NewReader(requestByte)
 	result := shared.Label{}
-	return apiClient.REST(repo.RepoHost(), "POST", path, requestBody, &result)
+	err = apiClient.REST(repo.RepoHost(), "POST", path, requestBody, &result)
+
+	if opts.Force {
+		if httpError, ok := err.(api.HTTPError); ok && httpError.StatusCode == 422 {
```

Comment:
Unfortunately this is not specific enough test to determine that the label create failed due to the label name already existing. The API will send back a 422 in at least two other cases that I found, when the color is not a valid hexadecimal color code, and when the description is more than 100 characters. I think this means that we will need to dig into the error more and verify that it is the name field that is causing the error.

Reply:
Additionally, what are your thoughts on having this function return a specific `labelAlreadyExistsError` so that `createRun` can give the user a nice hint to use the new `--force` flag if they want.

Reply:
Ended up using `errLabelAlreadyExists` or I'd get a static analyzer warning for not following the `errFoo` pattern.